shared/utils/combine-ovals.py

```diff
@@ -8,6 +8,7 @@
 import re
 import sys
 from copy import deepcopy
+from decimal import Decimal
```

Comment:
use tuples and str.split instead of Deciaml for version comparison. it works even for versions with 3 components

e.g.: `(5, 10) < (5, 11)`